### PR TITLE
Fix sizing bug

### PR DESCRIPTION
**Pull request by Wei-Xinn**
## Changes
This PR fixes the sizing issue for the onboarding window on linux.

### CODE REVIEW

File: src/main/resources/view/onboarding/OnboardingPrompt.fxml [hunk 1]
```diff
@@ -1,18 +1,14 @@
...
-<?import javafx.scene.layout.StackPane?>
-<?import javafx.scene.layout.VBox?>
-<?import javafx.stage.Stage?>
+<?import java.net.*?>
+<?import javafx.geometry.*?>
+<?import javafx.scene.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.image.*?>
+<?import javafx.scene.layout.*?>
+<?import javafx.stage.*?>
```
Comment (sharpstorm) [inline src/main/resources/view/onboarding/OnboardingPrompt.fxml:9]: Please revert wildcards.

**Comment by dandaandaaaaaan** [comment]
failing CI also remember to check

File: src/main/resources/view/onboarding/OnboardingPrompt.fxml (hunk 1 above)
Comment (Wei-Xinn) [reply src/main/resources/view/onboarding/OnboardingPrompt.fxml]: Fixed

**Comment by KwanHW** [comment]
If there is a merge conflict for the UI segments, take this PR instead as the debugging annotations earlier was persisted in #324

**Comment by dandaandaaaaaan** [comment]
I told him to just piggyback this PR to fix https://github.com/AY2122S2-CS2103-W17-1/tp/issues/272 since it is a small fix

File: src/main/resources/view/onboarding/OnboardingPrompt.fxml [hunk 2]
```diff
@@ -1,58 +1,57 @@
...
-    </icons>
-    <scene>
-        <Scene>
-            <stylesheets>
-                <URL value="@../DarkTheme.css" />
-            </stylesheets>
-            <VBox>
-                <children>
-                    <StackPane prefHeight="14.0" prefWidth="530.0" styleClass="pane-with-border" VBox.vgrow="ALWAYS">
+<fx:root maxHeight="400" maxWidth="700" minHeight="150" minWidth="500" resizable="true" title="Prompt" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/11" xmlns:fx="http://javafx.com/fxml/1">
```
Comment (sharpstorm) [inline src/main/resources/view/onboarding/OnboardingPrompt.fxml:14]: Please revert to JavaFX 8.
```suggestion
<fx:root maxHeight="400" maxWidth="700" minHeight="150" minWidth="500" resizable="true" title="Prompt" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
```

**Review by sharpstorm — APPROVED** [review]
LGTM.

**Review by dandaandaaaaaan — APPROVED** [review]
lgtm

**Pull request merged**